// 2016 misc / generator throw() caught by inner generator
module.exports = () => {
  function * generator() {
    yield * (function * () {
      try {
        yield 'foo';
      }
      catch(e) {
        return;
      }
    }());
    yield 'bar';
  }
  var iter = generator();
  iter.next();
  return iter['throw']().value === 'bar';

};
----------------------------------------------------------
Process spawn error. Is java in the path?
spawn /tmp/faast/08f51849-687f-49b0-bc1a-15cc2e88e7f3/node_modules/google-closure-compiler-linux/compiler EACCES
